docs/iot-platform-sdk/running-the-demo.md

```diff
@@ -508,9 +572,9 @@ Given an OwnershipVoucher file with a unique deviceId/GUID, the following needs
 
 The following is a sample Python\* script that, given an Owner voucher file as an argument, extracts the GUID and creates the previously mentioned device directory structure.
 
-This sample script operates under the assumption that both the script and voucher files are placed at \<sdo-iot-platform-sdk-root>/demo/ocs/config/db/v1/devices, and that this is the current working directory:
+This sample script operates under the assumption that both the script and voucher files are placed at <sdo-iot-platform-sdk-root>/demo/ocs/config/db/v1/devices, and that this is the current working directory:
```

Comment:
enclose the files location in `file_location` as is done on line 462 in the PR.
Please apply this to all the file location mentioned in this PR

Reply:
I will enclose the paths with file_location attribute.